web/src/components/Toolbar.tsx

```diff
@@ -21,42 +18,45 @@ const ToolbarContainer = styled.div<{ direction: 'row' | 'column'; hasPadding: b
     opacity: 1;
   }
 
+  & p {
+    margin: 0.5rem 0 0 0;
+  }
+
   @media ${dimensions.smallViewport} {
     width: 100%;
     flex-flow: row wrap;
     justify-content: center;
   }
 `
 
-const Headline = styled.h5`
-  width: 100vw;
-  margin: 0;
-  text-align: center;
-  font-size: 90%;
+const Divider = styled.hr`
+  margin: 12px 24px;
+  background-color: ${props => props.theme.colors.poiBorderColor};
```

Comment:
I think we should change the naming of the color `poiBorderColor` to something more generic